README.md

```diff
@@ -2208,25 +2217,32 @@ Other Style Guides
     > Why? It provides precision by distinguishing null/undefined from other falsy values, enhancing code clarity and predictability.
 
     ```javascript
-    // bad
-    const value = 0 ?? 'default';
-    // returns 0, not 'default'
-
-    // bad
-    const value = '' ?? 'default';
-    // returns '', not 'default'
+    // good
+    const value = '' ?? 'default'; // '', not 'default'
 
     // good
-    const value = null ?? 'default';
-    // returns 'default'
+    const value = null ?? 'default'; // 'default'
 
     // good
     const user = {
       name: 'John',
       age: null
     };
-    const age = user.age ?? 18;
-    // returns 18
+    const age = user.age ?? 18; // 18
+
+    // good
+    const user = {
+      name: 'John',
+    };
+    const age = user.age ?? 18; // 18
+
+    // good
+    const user = {
+      name: 'John',
+      age: 0
+    };
+    const age = user.age ?? 18; // 0
+    const defaultAge = user.age || 18 // 18
```

Comment:
This better demonstrates difference between || and ??

Reply:
yes, i like this.